### PR TITLE
[ci] Skip FPGA/ OTBN tests for CDC

### PR DESCRIPTION
**Pull request by eunchan**
This commit let CI skip FPGA, verilator, OTBN tests for CDC only
changes.

The cdconlychange log has been checked after
https://github.com/lowRISC/opentitan/pull/13117 has been merged.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -259,6 +259,7 @@ jobs:
 - job: otbn_standalone_tests
   displayName: Run OTBN Smoke Test
   dependsOn: lint
+  condition: and(succeeded(), eq(dependencies.lint.outputs['DetermineBuildType.onlyCdcChanges'], '0'))
```
Comment (eunchan) [inline azure-pipelines.yml:262]: @GregAC Just added this to skip OTBN. Other changes are amended to the pre `condition:` field.

**Comment by milesdai** [comment]
Could you point me to some of the logs that show that the script is working? I took a look around on the latest version of master but I couldn't see any printouts (e.g. https://dev.azure.com/lowrisc/opentitan/_build/results?buildId=83308&view=logs&j=3a559e2a-952e-58d2-b8db-2e604a9266d7&t=dcdf62d6-c818-5cd7-fbbc-5e277ab47ca1)

**Comment by eunchan** [comment]
> Could you point me to some of the logs that show that the script is working? I took a look around on the latest version of master but I couldn't see any printouts (e.g. https://dev.azure.com/lowrisc/opentitan/_build/results?buildId=83308&view=logs&j=3a559e2a-952e-58d2-b8db-2e604a9266d7&t=dcdf62d6-c818-5cd7-fbbc-5e277ab47ca1)

https://dev.azure.com/lowrisc/opentitan/_build/results?buildId=83335&view=logs&jobId=3a559e2a-952e-58d2-b8db-2e604a9266d7&j=3a559e2a-952e-58d2-b8db-2e604a9266d7&t=dcdf62d6-c818-5cd7-fbbc-5e277ab47ca1

**Review by milesdai — APPROVED** [review]

**Comment by eunchan** [comment]
@GregAC Could you please check and approve if it is OK?

**Review by GregAC — APPROVED** [review]

**Pull request merged**